roles/gcp/compute_instance/README.md

```diff
@@ -1,36 +1,47 @@
 GCP Compute Instance
 ====================
-[![Galaxy]()](https://galaxy.ansible.com/cloud.FIXME)
-[![Build Status](FIXME)
 
-A brief description of the role goes here.
+Manage compute instance(s) on GCP with the given requirements.
 
 Requirements
 ------------
 
-The google.cloud collection
+* The `google.cloud` collection -- installable via `ansible-galaxy collection install google.cloud`
 
 Role Variables
 --------------
 
+All variables can be defined in `defaults/main.yml` or overriden at the import of the role. **ALL** of the below values must be defined.
+
 | Name              | Default Value       | Description          |
 |-------------------|---------------------|----------------------|
-| `` | `` |  |
-
+| `gcp_project`     | `cloud-role` |  The name of previously created GCP project that must be available |
+| `gcp_cred_kind`     | `serviceaccount` |  Specifies what type of credential being used |
+| `gcp_cred_file`     | `~/auth.json` |  Specifies what credential file to be use. The credentials must have appropriate permissions to modify all necessary objects in the tasks. |
+| `gcp_region`     | `asia-south2` |  The region to deploy objects to |
+| `nodes`     | `` |  Details about the nodes to be deployed |
 
 Dependencies
 ------------
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
+N/A
 
 Example Playbook
 ----------------
 
-    - hosts: all
-      roles:
-         - ansibleproject.compute_instance # FIXME
+  ---
+  - hosts: localhost
+    collections:
+      - cloud.roles
+      - google.cloud
```

Comment:
This can be added in the ``meta/main.yml`` file from the role

```
collections:
  - google.cloud
```

Reply:
ok.